tests/codebuild/create_tests.sh

```diff
@@ -200,8 +200,9 @@ function verify_retain_varient_properties(){
 #    $4: Filename of the custom metric hap test to run
 function run_hap_test()
 {
+  local random_string=$RANDOM
   local target_namespace="$1"
-  local hosting_deployment_1="$2"
+  local hosting_deployment_1="${2}-${random_string}"
```

Comment:
line 232 in this method resets the endpoint name in the yaml file after the resources are applied. we need to use a different variable for that now?

also where is $RANDOM generated

Reply:
Since we are using `yq` everywhere now and not `sed` the actual value in these fields should not matter. However, I will make sure to parse the logs of a test run (including the namespace tests) to make sure everything actually ran as expected.